plugins/email_service/app/controllers/email_service/domain_verifications_controller.rb

```diff
@@ -0,0 +1,138 @@
+module EmailService
+  class DomainVerificationsController < ::EmailService::ApplicationController
+    before_action :restrict_access
```

Comment:
How you enforce your policy? I couldn't find any reference to the 
```
authorization_context 'email_service'
authorization_required
```

Reply:
done.